Got the error below? Did I have to run configure again? Or just make install?

make  all-am
make[3]: Entering directory `/root/rinatg/src'
g++ -DHAVE_CONFIG_H -I.  -pthread -I/usr/local/irati/include -I/usr/include/libnl3 -I/usr/local/include   -Wall -Wformat -Wnon-virtual-dtor -Woverloaded-virtual -Wextra -Werror -Wfatal-errors -pthread -I/usr/include -I./tclap  -Wall -Wformat -Wnon-virtual-dtor -Woverloaded-virtual -Wextra -Werror -Wfatal-errors -g -O2 -MT rina_tgen-simple_ap.o -MD -MP -MF .deps/rina_tgen-simple_ap.Tpo -c -o rina_tgen-simple_ap.o`test -f 'simple_ap.cc' || echo './'`simple_ap.cc
mv -f .deps/rina_tgen-simple_ap.Tpo .deps/rina_tgen-simple_ap.Po
g++ -DHAVE_CONFIG_H -I.  -pthread -I/usr/local/irati/include -I/usr/include/libnl3 -I/usr/local/include   -Wall -Wformat -Wnon-virtual-dtor -Woverloaded-virtual -Wextra -Werror -Wfatal-errors -pthread -I/usr/include -I./tclap  -Wall -Wformat -Wnon-virtual-dtor -Woverloaded-virtual -Wextra -Werror -Wfatal-errors -g -O2 -MT rina_tgen-server.o -MD -MP -MF .deps/rina_tgen-server.Tpo -c -o rina_tgen-server.o`test -f 'server.cc' || echo './'`server.cc
mv -f .deps/rina_tgen-server.Tpo .deps/rina_tgen-server.Po
g++ -DHAVE_CONFIG_H -I.  -pthread -I/usr/local/irati/include -I/usr/include/libnl3 -I/usr/local/include   -Wall -Wformat -Wnon-virtual-dtor -Woverloaded-virtual -Wextra -Werror -Wfatal-errors -pthread -I/usr/include -I./tclap  -Wall -Wformat -Wnon-virtual-dtor -Woverloaded-virtual -Wextra -Werror -Wfatal-errors -g -O2 -MT rina_tgen-client.o -MD -MP -MF .deps/rina_tgen-client.Tpo -c -o rina_tgen-client.o`test -f 'client.cc' || echo './'`client.cc
mv -f .deps/rina_tgen-client.Tpo .deps/rina_tgen-client.Po
make[3]: *** No rule to make target`timeutils.cc', needed by `rina_tgen-timeutils.o'.  Stop.
make[3]: Leaving directory`/root/rinatg/src'
make[2]: **\* [all] Error 2
make[2]: Leaving directory `/root/rinatg/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory`/root/rinatg'
make: **\* [all] Error 2

--- a/src/main.cc
+++ b/src/main.cc
@@ -216,14 +216,14 @@ int main(int argc, char * argv[])
                         if (distribution_type == "CBR" || distribution_type == "cbr")
                                 c.single_cbr_test(size,
                                                 count,
-                                                duration,
+                                                duration*1000,
                                                 rate,
                                                 busy,
                                                 port_id);
                         else if (distribution_type == "poisson")
                                 c.single_poisson_test( size,
                                                        count,
-                                                       duration,
+                                                       duration*1000,
                                                        rate,
                                                        busy,
                                                        poisson_mean,

